src/components/AppComponents/AppHeaderBurgerMenuSections/AppHeaderBurgerMenuSections.tsx

```diff
@@ -27,15 +28,15 @@ const AppHeaderBurgerburgerMenuSections: FC<
   AppHeaderBurgerburgerMenuSectionsProps
 > = (props) => {
   const { burgerMenuSections } = props;
-  const { user } = useUser();
+  const clerk = useFeatureFlaggedClerk();
 
   return (
     <OakFlex $flexDirection="column" $gap="all-spacing-7">
-      {user && (
-        <OakSecondaryButton element="a" href="/api/auth/logout">
-          Sign out
-        </OakSecondaryButton>
-      )}
+      <clerk.SignedIn>
+        <clerk.SignOutButton>
+          <OakSecondaryButton>Sign out</OakSecondaryButton>
+        </clerk.SignOutButton>
+      </clerk.SignedIn>
```

Comment:
Until we remove the feature flag all Clerk components should be access through the `useFeatureFlaggedClerk` hook to ensure they are not rendered for users without the flag enabled.